index.html

```diff
@@ -49,6 +49,7 @@
     <div id='nav'>
       <div class='container center'>
         <a href="#contribute">Contribute</a>
+        <a href="#ci">CI</a>
```

Comment:
Yeah I think I'd rather make CI more discoverable than Memory Calculator and Plan Checker... maybe a new section called "Tools" and put those 2 things there and let CI have it's own entry? Sorry to slightly thread-jack this.